### DISCUSSION TITLE
Best method of conversion?

### DISCUSSION THREAD

**Discussion by AppleFritter58** [lukas-blecher/LaTeX-OCR#255] [Q&A]
Using the first method `pix2tex` I converted this:

![image](https://user-images.githubusercontent.com/100169858/231839217-8cc34e29-16bf-4823-b08e-19cc6ec81c3c.png)

Into this:

$\textstyle{\oint_{\alpha}^{\infty}\varphi\left(\mathbf{d}\gg\mathrm{d}\Omega\right)}$

Obviously this isn't ideal. What's the best method to make latex with the accuracy mathpix has?

**Comment by lukas-blecher** [comment]
The pretrained model is not trained on handwritten formulas, only printed. Somebody would need to finetune or train the model on handwritten equations as well.
